docker/Dockerfile

```diff
@@ -77,6 +77,9 @@ RUN ln -s /var/lib/runit/service/nginx /etc/service/nginx
 ADD dist /opt/spark/dist
 ADD krb5.conf.mustache /etc/
 
+# Add external jar for pushing metrics via StatsD (https://github.com/vidhyaarvind/spark-statsd)
+ADD jars/spark-statsd-1.0.0.jar /opt/spark/dist/jars/
```

Comment:
Would it make sense to include this from `downloads.mesosphere` instead, or is there precedent  to add it like this. What worries me is that there is no indication as to how the jar was built.

Reply:
Not sure about precedence but I do think this is easier (since we don't have an equivalent SDK `svc.yml` file). However the link in the comment does point to the project with instructions to build the jar.

If we went the `downloads.mesosphere` route: 
- we would need to add a file (or files) to build the jar
- add a jenkins job to build and upload the jar to the mesosphere acct 
- update Spark's marathon cmd to mv the file to directory (`opt/spark/dist/jars`)
- (not sure if there are more steps beyond that)

Not saying it's hard but it would add more work for the same result. Not sure its worth the effort?